srcpkgs/gdb/template

```diff
@@ -1,65 +1,60 @@
  $(vopt_if static 'CFLAGS=-static CXXFLAGS=-static LDFLAGS=-static')
- $(vopt_if python --with-python=/usr/bin/python3)
+ $(vopt_with guile) $(vopt_if python --with-python=/usr/bin/python3)
  $(vopt_with debuginfod)"
-hostmakedepends="texinfo perl pkg-config $(vopt_if python python3-devel)"
-makedepends="ncurses-devel zlib-devel readline-devel expat-devel
- $(vopt_if python 'python3-devel gettext-devel')
- $(vopt_if debuginfod elfutils-devel)"
+hostmakedepends="pkg-config texinfo $(vopt_if python python3-devel) $(vopt_if guile guile)"
+makedepends="expat-devel ncurses-devel readline-devel zlib-devel $(vopt_if guile guile-devel)
+ $(vopt_if python 'python3-devel gettext-devel') $(vopt_if debuginfod elfutils-devel)"
+depends="gdb-common-${version}_${revision}"
+checkdepends="dejagnu"
 short_desc="GNU Debugger"
 maintainer="Anthony Iliopoulos <ailiop@altatus.com>"
 license="GPL-3.0-or-later"
-homepage="https://www.gnu.org/software/gdb/"
-distfiles="${GNU_SITE}/${pkgname}/${pkgname}-${version}.tar.xz"
+homepage="https://www.gnu.org/software/gdb"
+distfiles="${GNU_SITE}/gdb/gdb-${version}.tar.xz"
 checksum=f82f1eceeec14a3afa2de8d9b0d3c91d5a3820e23e0a01bbb70ef9f0276b62c0
+make_check=extended  # Tests take too long, not all of them pass.
 python_version=3
 patch_args="-Np1"
+ignore_elf_files="
+ /usr/share/gdb/guile/gdb/support.go
+ /usr/share/gdb/guile/gdb/experimental.go
+ /usr/share/gdb/guile/gdb/iterator.go
+ /usr/share/gdb/guile/gdb/types.go
+ /usr/share/gdb/guile/gdb/printing.go
+ /usr/share/gdb/guile/gdb.go"
 
 if [ "${CROSS_BUILD}" ]; then
 	# Make python3.x detection work in cross builds
 	CFLAGS="-I${XBPS_CROSS_BASE}/${py3_inc}"
 	CXXFLAGS="-I${XBPS_CROSS_BASE}/${py3_inc}"
+	makedepends+=" libatomic_ops-devel" # guile
 fi
-CFLAGS+=" -fcommon"
-CXXFLAGS+=" -fcommon"
+
 # Package build options
-build_options="gdbserver static python debuginfod"
+build_options="debuginfod gdbserver guile python static"
 desc_option_gdbserver="Enable support for building GDB server"
 desc_option_debuginfod="Enable support for libdebuginfod"
 # Enable gdbserver if !static.
 build_options_default="gdbserver python debuginfod"
 # Both options cannot be enabled at the same time
 vopt_conflict gdbserver static
 
-post_extract() {
-	vsed -i 's,sgidefs.h,asm/sgidefs.h,' gdb/mips-linux-nat.c
-	mkdir -p build
-}
-do_configure() {
-	cd build
-	../configure ${configure_args/with-sysroot/with-build-sysroot} \
-		CFLAGS_FOR_BUILD="${CFLAGS_FOR_BUILD} -fcommon"
-	export gl_cv_func_gettimeofday_clobber=no
-	export gl_cv_func_working_strerror=yes
-	export gl_cv_func_strerror_0_works=yes
-}
-do_build() {
-	cd build
-	make ${makejobs} all
-}
-do_install() {
-	cd build
-	make DESTDIR=${DESTDIR} ${makejobs} install
+post_install() {
 	# resolve conflicts with binutils
-	rm -rf ${DESTDIR}/usr/{include,lib,lib64}
-	for f in bfd configure standards; do
-		rm -f ${DESTDIR}/usr/share/info/${f}.info*
-	done
+	rm -rf ${DESTDIR}/usr/{include,lib,share/info/bfd.info*}
```

Comment:
Why not remove `configure` and `standards`?

Also, if possible, use `rm -r` instead of `-rf`

Reply:
> Why not remove configure and standards?

These files just don't exist.